src/layouts/components/Header.tsx

```diff
@@ -143,7 +160,24 @@ export function Header(): JSX.Element {
   );
 }
 
-function DesktopNavbar(): JSX.Element {
+async function getOpenProposals(api: WhaleApiClient) {
```

Comment:
are we still keeping this variable name?

Reply:
- let's move this function to within `Header` component
- and then let's use `WhaleApiClient` right inside the function, should be a param